masterlist.yaml

```diff
@@ -6769,6 +6769,47 @@ plugins:
       # version: 1.06a
       - crc: 0x20AA5984
         util: 'SSEEdit v3.2.2'
+  - name: 'Immersive Movement.esp'
+    group: *lowPriorityGroup
```

Comment:
This group assignment isn't necessary and will cause sorting errors.

Reply:
Removed.